### ISSUE TITLE
[Bug?] A switch connected to another switch doesn't mirror states

### ISSUE THREAD

**Issue by captain828** [tsunamayo/Starship-EVO#1889]
Since I don't really know if this should be considered a bug or not, I won't add too much info.

### Given the following scenario:
- we have two switches, A and B
- switch A is connected to switch B
- switch B is connected to a light L
- both switches are initially in the OFF position

### When:
1. switch A is turned ON:
- switch B also turns ON
- light L turn green
2. after 1), switch A is turned OFF:
- switch B stays ON
- light L remains green
3. after 2), switch A is turned ON:
- switch B turns OFF
- light L turns red

### Expected behavior:
- after 2), switch B should turn OFF and light L turn red
- after 3), switch B should turn ON and light L turn green
- so basically a switch connected to another switch should mirror its state

I understand why it's coded this way (binary switch) but IMO it's not intuitive in-game.

**Comment by Battlepixel** [comment]
It s no bug. You have to connect a button with a switch and use an "indicator ligth" to show on/off-state

**Comment by tsunamayo** [comment]
Yes, it was following a discussion and suggestion - I am not sure this is the right thing to do, it is indeed a bit complicated to understand.

**Comment by Xerma** [comment]
I think the way it works in-game is perfectly fine honestly. To receive your desired function you could even link a button to that switch. The way it works is the switch sends and holds a 1 while on so when a switch receives a signal from another switch it basically just works like a simple toggle

**Comment by captain828** [comment]
I'm personally fine with it working as it currently does so if you think this shouldn't change then feel free to close the issue.
